js/src/services/vue.service.js

```diff
@@ -1,17 +1,36 @@
 import Vue from 'vue';
 import SuiVue from 'semantic-ui-vue';
-
 import atkClickOutside from '../directives/click-outside.directive';
 import { focus } from '../directives/commons.directive';
 
 Vue.use(SuiVue);
 
+// Vue loader component to display while dynamic component is loading.
+const atkVueLoader = {
+    name: 'atk-vue-loader',
+    template: '<div><div class="ui active centered inline loader"></div></div>',
```

Comment:
I see it is empty, but why this at all?

Reply:
Yes, in case a component is too large and take some time to load.
We can see it sometime using query builder